data.tf

```diff
@@ -28,6 +28,23 @@ data "aws_ami" "eks_worker" {
   owners = ["602401143452"]
 }
 
+####################################################
+# Fetch the latest ADSK SAMI for EKS
+####################################################
+# Version number should not contain the period. For example: 1.15 -> 115
+
+data "aws_ami" "sami" {
+  filter {
+    name   = "name"
+    values = ["SAMI-Linux-Amazon-2-EKS-${replace(var.cluster_version, ".", "")}-*"]
+  }
+
+  most_recent = true
+
+  # Owner ID of ADSK SAMI
+  owners = ["996087537883"]
```

Comment:
Maybe a silly question, but what is this number? Also totally a nit, but curious if it'd be better to have something like this in a variable rather than hard-coded here. 🤷‍♀️  Just a thought, not sure if it matters.

Reply:
This is the account ID that owns the Autodesk SAMI. It shouldn't change but making it a variable is a good idea.